testsuite/features/secondary/change_software_channel.feature

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2020 SUSE LLC
```

Comment:
Title of feature does not follow conventions. Prefix it with `min_` please.

Reply:
changed